boards/arm/frdm_k64f/Makefile.board

```diff
@@ -1,8 +1,5 @@
-FLASH_SCRIPT = openocd.sh
-DEBUG_SCRIPT = openocd.sh
-
```

Comment:
Do not remove those, flashing using openocd works with openocd 0.10, so lets keep that, I am fine changing the default to pyocd right now if @MaureenHelm  is ok with that.